public/app/features/dashboard-scene/inspect/InspectJsonTab.test.tsx

```diff
@@ -265,24 +255,13 @@ async function buildTestSceneWithLibraryPanel() {
 
   jest.spyOn(libpanels, 'getLibraryPanel').mockResolvedValue({ ...libraryPanelState, ...panel });
 
-  const gridItem = new DashboardGridItem({
-    key: 'griditem-1',
-    x: 0,
-    y: 0,
-    width: 10,
-    height: 12,
-    body: libraryPanel,
-  });
-
   const scene = new DashboardScene({
     title: 'hello',
     uid: 'dash-1',
     meta: {
       canEdit: true,
     },
-    body: new SceneGridLayout({
-      children: [gridItem],
-    }),
+    body: DefaultGridLayoutManager.fromVizPanels([libraryPanel]),
```

Comment:
this DefaultGridLayoutManager.fromVizPanels is a util function that is only used from tests, we have a ton of tests that does not really care about defining the grid items and their positions, so this util function does that automatically